webapp/config/settings/base.py

```diff
@@ -158,6 +158,7 @@
         'dj_rest_auth.jwt_auth.JWTCookieAuthentication',
         'rest_framework_simplejwt.authentication.JWTAuthentication',
     ),
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination'
```

Comment:
```suggestion
    'DEFAULT_PAGINATION_CLASS': 'config.pagination.PageNumberPagination'
```
p1: 위와 같이 되어야하지 않을까요?

Reply:
수정했습니다.